nexus/src/external_api/http_entrypoints.rs

```diff
@@ -5640,15 +5640,17 @@ async fn vpc_subnet_update(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
-/// List network interfaces
+/// List network interfaces for a VPC subnet
+/// Use `/v1/vpc-subnets/{subnet}/network-interfaces` instead
 #[endpoint {
     method = GET,
     path = "/organizations/{organization_name}/projects/{project_name}/vpcs/{vpc_name}/subnets/{subnet_name}/network-interfaces",
     tags = ["vpcs"],
+    deprecated = true,
 }]
 async fn vpc_subnet_list_network_interfaces(
     rqctx: RequestContext<Arc<ServerContext>>,
-    query_params: Query<PaginatedByName>,
+    query_params: Query<PaginatedByNameOrId>,
```

Comment:
This is fine, but you didn't have to change the pagination type for the old endpoint. There are other examples of the pagination style being updated under the hood but keeping the endpoint the same. 

Honestly, it doesn't matter here given we're going to delete this anyway and it's not a breaking change to the existing API.

Reply:
Yeah, I figured this was actually slightly easier.